shared/languages.js

```diff
@@ -2,9 +2,54 @@ import ISO6391 from 'iso-639-1';
 
 export const ENGLISH_CODE = 'en';
 
-export const getAllLanguages =
-  () => ISO6391.getLanguages(ISO6391.getAllCodes());
+const ADDITIONAL_LANGUAGES = [
+  {
+    code: 'zh-CN',
+    name: 'Chinese - China',
+    nativeName: '中文 (中国)',
+  },
+  {
+    code: 'zh-TW',
+    name: 'Chinese - Taiwan',
+    nativeName: '中文 (台灣)',
+  },
+  {
+    code: 'zh-HK',
+    name: 'Chinese - Hong Kong',
+    nativeName: '中文 (香港)',
+  },
+];
 
-export const getLanguages = ISO6391.getLanguages.bind(ISO6391);
+const LANGUAGES_TO_REMOVE = [
+  'zh',
+];
 
-export const getLanguageName = ISO6391.getNativeName.bind(ISO6391);
+export const getAllLanguages = () => {
+  const isoLanguages = ISO6391.getLanguages(ISO6391.getAllCodes());
+  const allLanguages = changeLanguages(isoLanguages);
+  return allLanguages;
+};
+
+export const getLanguages = (codes) => {
+  const languages = ISO6391.getLanguages(codes);
+  return languages;
+};
+
+export const getLanguageName = (code) => {
+  const language = getAllLanguages().find((language) => language.code === code);
```

Comment:
Not sure this is the best strategy, but I guess it makes the impact of `LANGUAGES_TO_REMOVE` and `ADDITIONAL_LANGUAGES` the smallest.